src/api/services/ItemPriceService.ts

```diff
@@ -44,22 +48,22 @@ export class ItemPriceService {
         const body = JSON.parse(JSON.stringify(data));
 
         const shippingPrice = body.shippingPrice;
-        const cryptocurrencyAddress = body.cryptocurrencyAddress;
+        const cryCurAddress = body.cryptocurrencyAddress;
 
         delete body.shippingPrice;
         delete body.cryptocurrencyAddress;
 
         // create related models, cryptocurrencyAddress
-        if (!_.isEmpty(cryptocurrencyAddress)) {
-            if (cryptocurrencyAddress.id) {
+        if (!_.isEmpty(cryCurAddress)) {
```

Comment:
can you change these back, cryCurAddress is not at all as readable as cryptocurrencyAddress.